common/components/WalletDecrypt/components/LedgerNano.tsx

```diff
@@ -58,14 +49,14 @@ class LedgerNanoSDecryptClass extends PureComponent<Props, State> {
 
   public render() {
     const { network } = this.props;
-    const { dPath, publicKey, chainCode, error, isLoading, showTip } = this.state;
+    const { dPath, publicKey, chainCode, error, isLoading } = this.state;
     const showErr = error ? 'is-showing' : '';
 
     if (!dPath) {
       return <UnsupportedNetwork walletType={translateRaw('x_Ledger')} />;
     }
 
-    if (window.location.protocol !== 'https:') {
+    if (!process.env.BUILD_ELECTRON && window.location.protocol !== 'https:') {
```

Comment:
We use this env variable everywhere in the codebase, perhaps use another level of indirection by hiding it behind a function like `isElectron` so we can change it if need be in the future? We could do this for any other environment variables too.